packages/worker/src/worker.ts

```diff
@@ -764,10 +811,18 @@ export class Worker {
           return { activation, parentSpan };
         });
       } catch (err) {
-        parentSpan.setStatus({ code: otel.SpanStatusCode.ERROR }).end();
+        if (!(err instanceof errors.ShutdownError)) {
```

Comment:
Looks like [instrument](https://github.com/temporalio/sdk-typescript/blob/ce23c2c96b3e9bbaa46378d5658f6698d3f6bd5f/packages/worker/src/tracing.ts#L28) already marks the span with code: ERROR

Reply:
So are you suggesting we invert this to un-set it?

Reply:
I don't know if this is possible, we would probably need to make shutdown error not an `Error`, maybe instead we can return an optional value and break the loop if it's undefined.